node.js_mongodb/bulkupload.js

```diff
@@ -0,0 +1,30 @@
+const DBHelper=require('./DB_helper');
+const readline = require('readline');
+const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout
+  });
+  console.log("Attempting to upload all files in '/data' directory to mongoDB cloud.")
+  rl.question('Are you sure you want to upload and overwrite all data? (Y/N) ', async (reply) => {
+      if(reply.indexOf("Y")>=0){
+         await bulkUpload();
+      }
+      rl.close();
+  });
+
+async function bulkUpload(){
```

Comment:
Is this bulkUpload any different from the bulkUpload() func in app.js? Do we need both?